### PR TITLE
#106 AAU I can see a new page with the description and synopsis wI click on a book in the list

### PR DESCRIPTION
**Pull request by howant**

### CODE REVIEW

File: frontend/src/components/Description.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React, { Component } from "react";
+
+class Description extends Component {
+
+  render() {
+    return (
+      <div>
+        <h2>Description goes here</h2>
```
Comment (howant) [inline frontend/src/components/Description.js:8]: Description to be updated.

File: frontend/src/App.js [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
-        </header>
-        <Booklist/>
-      </div>
+      <Router>
+        <div className="App">
+          <header className="App-header">
+            <h1 className="App-title">My Book List</h1>
+          </header>
+          <Route exact path="/" component={Booklist} />
+          <Route path="/description" component={Description} />
```
Comment (robcronin) [inline frontend/src/App.js:17]: If you add the following at the bottom then any bad routes will render the main page
```
<Route path="/description" component={Booklist} />
```

**Review by robcronin — CHANGES_REQUESTED** [review]

File: frontend/src/App.js (hunk 2 above)
Comment (howant) [reply frontend/src/App.js:17]: added:
<Route path="/" component={Booklist} />

File: frontend/src/components/Description.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+      </div>
+    );
+  }
+}
+
+export default Description;
```
Comment (robcronin) [inline frontend/src/components/Description.js:14]: You should add a snapshot test for your new component. Ideally every component would have a snapshot

**Review by robcronin — CHANGES_REQUESTED** [review]

File: frontend/src/components/Description.js (hunk 3 above)
Comment (howant) [reply frontend/src/components/Description.js:14]: Description test added. Jest tests moved to test folder.

**Review by robcronin — APPROVED** [review]

**Pull request merged**